tests/CommandsTest.php

```diff
@@ -94,6 +94,23 @@
     expect(Schema::hasTable('users'))->toBeTrue();
 });
 
+test('migrate command does not throw exception if tenant does not have a database and skip-failing is passed', function() {
```

Comment:
Since this test also asserts that the exception *is* thrown in the opposite case, the name should be something like "migrate command only throws exceptions if skip-failing is not passed"? So that it covers both cases.

Reply:
Yeah, that name is much more concise too. I'll use it.